### ISSUE TITLE
End to Procrastination.

### ISSUE THREAD

**Issue by sharmaanj200** [alexcg1/good-first-issues#33]
Idea:
A website that tracks the time we have spent doing something productive and calculates the amount of time we have wasted, which could have been spent better. Helps us make timetables and provides rewards or bonus points based on how many tasks we have completed and deducts points if any assigned task is not finished within the deadline. Helps us focus better and provides various tips/tricks as rewards once the required task of the day is finished.

Tech stack:
ML Models to make efficient timetables, HTML, CSS, JAVASCRIPT, Bootstrap.

Motivation behind the idea:
I am a huge procrastinator and waste a large amount of my time thinking about how I can postpone stuff which leads me to waste a lot of time and not get done with anything which needs to be done. I think this website would help people like me who are wants to be productive but cannot follow through, ever.
